### DISCUSSION TITLE
Could there be an easier way to go back to notes from settings page?

### DISCUSSION THREAD

**Discussion by rishabhrathod01** [DenoSaurabh/space#4] [General]
Currently, when there are multiple collections in notes and a user clicks on the settings icon.

The only option to go back to that particular collection of notes is to click on notes in the navbar and then select the particular collection. The UX can be improved here.

Suggestion 
1. Either `settings` page could be a modal with the same UI and on close of which users stay where they were before opening settings.
Or 
2. Maybe each notes collection is a different page route and on close of settings, the user is redirected back.
For eg :- `/notes/collection1` (on click on settings) ->  `/settings` (on close)  -> `/notes/collection1`

**Comment by denosaurabh** [comment]
The app will be very personalizable & flexible, so the settings page will grow bigger and bigger. But, I liked the idea of routes for different collections, I also just implemented it. So this will be included in the next update, which will come around the end of this week.
